README.md

```diff
@@ -120,7 +126,7 @@ Example usage for `pip-sync`
 ============================
 
 Now that you have a `requirements.txt`, you can use `pip-sync` to update your
-virtual env to reflect exactly what's in there.  Note: this will
+virtual environment to reflect exactly what's in there.  Note: this will
 install/upgrade/uninstall everything necessary to match the `requirements.txt`
 contents.
```

Comment:
To make things explicit, you could add here «except for setuptools, pip and pip-tools itself.»